src/config.rs

```diff
@@ -0,0 +1,64 @@
+use secrecy::{ExposeSecret, Secret};
+use std::str::FromStr;
+
+#[derive(Clone, Debug, PartialEq)]
+pub enum PointsSource {
+    Path(String),
+    Link(String),
+}
+
+impl Default for PointsSource {
+    fn default() -> Self {
+        PointsSource::Path("".to_string())
+    }
+}
+
+/// Configuration for the EigenDA remote disperser client.
+#[derive(Clone, Debug, PartialEq, Default)]
+pub struct EigenConfig {
+    /// URL of the Disperser RPC server
+    pub disperser_rpc: String,
+    /// Block height needed to reach in order to consider the blob finalized
+    /// a value less or equal to 0 means that the disperser will not wait for finalization
+    pub settlement_layer_confirmation_depth: i32,
+    /// URL of the Ethereum RPC server
+    pub eigenda_eth_rpc: String,
+    /// Address of the service manager contract
+    pub eigenda_svc_manager_address: String,
```

Comment:
should we use https://docs.rs/alloy-primitives/latest/alloy_primitives/struct.Address.html instead of String?